test/checkpointctl.bats

```diff
@@ -589,3 +589,39 @@ function teardown() {
 	[ "$status" -eq 1 ]
 	[[ ${lines[0]} == *"no process with PID 9999"* ]]
 }
+
+@test "Run checkpointctl list with no checkpoints" {
+    checkpointctl list
+    [ "$status" -eq 0 ]
+    [[ ${lines[0]} == *"No checkpoints found"* ]]
+}
+
+@test "Run checkpointctl list with checkpoints in tar files" {
+    mkdir -p "$TEST_TMP_DIR1"/checkpoint-1
+    mkdir -p "$TEST_TMP_DIR1"/checkpoint-2
+
+    touch "$TEST_TMP_DIR2"/checkpoint-pod-name_namespace1-container-name-2023-10-12T11:02:56+01:00.tar
+    touch "$TEST_TMP_DIR2"/checkpoint-counter-tl58j_default-counter-2023-10-12T11:02:56+01:00.tar
+
+    checkpointctl list -p "$TEST_TMP_DIR2"
+    [ "$status" -eq 0 ]
+
+    [[ "${lines[4]}" =~ "checkpoint-counter-tl58j_default-counter-2023-10-12T11:02:56+01:00.tar" ]]
+    [[ "${lines[6]}" =~ "checkpoint-pod-name_namespace1-container-name-2023-10-12T11:02:56+01:00.tar" ]]
+
+    [[ "${lines[4]}" =~ "default" ]]
+    [[ "${lines[4]}" =~ "counter-tl58j" ]]
+    [[ "${lines[4]}" =~ "counter" ]]
+    [[ "${lines[4]}" =~ "12 Oct 23 11:02 +0100" ]]
+
+    [[ "${lines[6]}" =~ "namespace1" ]]
+    [[ "${lines[6]}" =~ "pod-name" ]]
+    [[ "${lines[6]}" =~ "container-name" ]]
+    [[ "${lines[6]}" =~ "12 Oct 23 11:02 +0100" ]]
```

Comment:
Since they are complete substrings being matched, it would be better to use `==` with a wildcard rather than a regex match, e.g. `[[ "${lines[4]}" == *"default"* ]]`